### PR TITLE
[Vectorized][Function] support regexp&&sm4&&aes functions 

### LINKED ISSUE #8306: [Vectorized][Feature] support regexp&&sm4&&aes function
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Description

_No response_

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhangstar333**
# Proposed changes

Issue Number: close #8306 

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/vec/functions/function_bitmap.cpp [hunk 1]
```diff
@@ -474,4 +474,4 @@ void register_function_bitmap(SimpleFunctionFactory& factory) {
     factory.register_function<FunctionBitmapSubsetInRange>();
 }
 
-} // namespace doris::vectorized
+} // namespace doris::vectorized
```
Comment (yangzhg) [inline be/src/vec/functions/function_bitmap.cpp:477]: new empty line

File: be/src/vec/functions/function_encryption.cpp [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+    DataTypes get_variadic_argument_types_impl() const override {
+        return Impl::get_variadic_argument_types_impl();
+    }
+
+    size_t get_number_of_arguments() const override {
+        return get_variadic_argument_types_impl().size();
+    }
+
+    bool use_default_implementation_for_nulls() const override { return false; }
```
Comment (HappenLee) [inline be/src/vec/functions/function_encryption.cpp:51]: why ```use_default_implementation_for_nulls``` is false

File: be/src/vec/functions/function_hex.cpp [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    static DataTypes get_variadic_argument_types() {
+        return {std::make_shared<vectorized::DataTypeHLL>()};
+    }
+
+    static Status vector(ColumnPtr argument_column, size_t input_rows_count,
+                         ColumnString::Chars& res_data, ColumnString::Offsets& res_offsets) {
+        const auto* str_col = check_and_get_column<ColumnHLL>(argument_column.get());
+        const auto& hll_data = str_col->get_data();
+        size_t total_length = 0;
+        std::vector<std::string> hll_str(input_rows_count);
```
[the remark this answers is not in the document]
Comment (zhangstar333) [reply be/src/vec/functions/function_hex.cpp]: done.

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**